### PR TITLE
Insert the column snoozed_at  on database

### PR DESCRIPTION
**Pull request by NFavareto**
**Description**

This PR create a new column named **snoozed_at** in to hellper's database.

**How to test?**

If you are going to using the local environment for this test, first run this code on the local database:

ALTER TABLE incident
  ADD snoozed_at TIMESTAMPTZ NULL;

**Expected behavior**

Hellper must be able to create the incident without a problem.

### CODE REVIEW

File: internal/model/sql/postgres/schema/hellper.sql [hunk 1]
```diff
@@ -19,6 +19,7 @@ CREATE TABLE public.incident (
 	severity_level int4 NULL,
 	channel_name text NULL,
 	updated_at timestamp NOT NULL DEFAULT now(),
+  snoozed_at timestamptz NULL,
```
Comment (fvbaltar) [inline internal/model/sql/postgres/schema/hellper.sql]: is it correct? or would it be "timestamp"
for this field is it important to evaluate the timezone ?

File: internal/model/sql/postgres/schema/hellper.sql [hunk 2]
```diff
@@ -19,6 +19,7 @@ CREATE TABLE public.incident (
 	severity_level int4 NULL,
 	channel_name text NULL,
 	updated_at timestamp NOT NULL DEFAULT now(),
+  snoozed_at timestamp NULL,
```
Comment (tiagoemsi) [inline internal/model/sql/postgres/schema/hellper.sql:22]:
```suggestion
  snoozed_at timestamptz NULL,
```

File: internal/model/sql/postgres/schema/hellper.sql (hunk 1 above)
Comment (NFavareto) [reply internal/model/sql/postgres/schema/hellper.sql]: [This commit](https://github.com/ResultadosDigitais/hellper/pull/84/commits/ba65079a7e164f040c479295e6982e611d8f5333) change to **timestamp**

File: internal/model/sql/postgres/schema/hellper.sql (hunk 2 above)
Comment (NFavareto) [reply internal/model/sql/postgres/schema/hellper.sql:22]: I am in doubt if we put the timestamp or timestamptz. :thinking:

File: internal/model/sql/postgres/schema/hellper.sql (hunk 1 above)
Comment (GLOFonseca) [reply internal/model/sql/postgres/schema/hellper.sql:22]: @tiagoemsi I don't know the best option here too, can you explain why use `timestamptz`?

Comment (tiagoemsi) [reply internal/model/sql/postgres/schema/hellper.sql:22]: The `timestamptz` data is the timestamp with time zone. The PostgreSQL provides two temporal data types for handling timestamp, one **without** timezone (`timestamp`) and one **with** timezone (`timestamptz`).

```sql
SET timezone = 'America/Los_Angeles';

SELECT ts, tstz FROM	timestampz_demo;

         ts          |          tstz
---------------------+------------------------
 2016-06-22 19:10:25 | 2016-06-22 19:10:25-07
(1 row)
```

I recommend using the same pattern:

```sql
	[...]
	start_ts timestamptz NULL,
	end_ts timestamptz NULL,
	identification_ts timestamptz NULL,
	[...]
```

Comment (tiagoemsi) [reply internal/model/sql/postgres/schema/hellper.sql:22]: We can use `timestamp`, no problem, but I also recommend changing the others

File: internal/model/sql/postgres/schema/hellper.sql (hunk 2 above)
Comment (tiagoemsi) [reply internal/model/sql/postgres/schema/hellper.sql]: I recommend using the same pattern:

```
	[...]
	start_ts timestamptz NULL,
	end_ts timestamptz NULL,
	identification_ts timestamptz NULL,
	[...]
```

File: internal/model/sql/postgres/schema/hellper.sql (hunk 1 above)
Comment (GLOFonseca) [reply internal/model/sql/postgres/schema/hellper.sql:22]: OK, I agreed in using either, but the same for all.

**Pull request merged**